server/package.json

```diff
@@ -18,7 +18,8 @@
     "pg-monitor": "^0.8.3",
     "pg-native": "^3.0.0",
     "pg-promise": "^7.0.3",
-    "postgrator": "^2.10.3"
+    "postgrator": "^2.10.3",
+    "query-string": "^6.13.6"
```

Comment:
query-string should probably be installed in the package.json for the react-ui to prevent confusion as it is used for the front end.